m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -1591,16 +1591,16 @@ PROCEDURE PRINTCURRENTSTATE
    PRINT INDENT(BASEINDENT+4)+Q+"sourceAccount"+Q+": "+Q+TRANSOURCEACCT(TMPLOOP)+Q+","
    NEWLINE
 
-
+   CHARCHECK=TRANACCOUNTNAME(TMPLOOP)
+   CALL CHECKFORINVALIDCHARS
+   TRANACCOUNTNAME(TMPLOOP)=CHARCHECKRETURN
```

Comment:
Maybe remove this line where you are assigning CHARCHECKRETURN back to the TRANACCOUNTNAME array?  Instead just reference CHARCHECKRETURN on the next line where you are printing the `accountName` property.

Reply:
@rrobison44 : I was actually thinking about that but I figured that updating the same variable with the corrected/modified value was the safest bet as otherwise, I'd have to go through the code after printing the value to verify that leaving the original (uncorrected/unmodified) variable value intact wouldn't be causing further issues downstream.